lib/index.ts

```diff
@@ -103,7 +104,6 @@ with the debug (-d) flag at \x1b[4mhttps://support.snyk.io/hc/en-us/requests/new
     );
     return {
       dependencyGraph: result.dependencyGraph,
-      package: 'n/a', // TODO: Will remove when everything is ported to depGraphs
```

Comment:
No, everything is not ported to `depGraphs`, but the CLI type had the field optional.